content/docs/instrumenting/exporters.md

```diff
@@ -89,22 +77,32 @@ hosted outside of the Prometheus GitHub organization.
    
 ### Other monitoring systems
    * [Cloud Foundry Firehose exporter](https://github.com/cloudfoundry-community/firehose_exporter)
-   * [scollector exporter](https://github.com/tgulacsi/prometheus_scollector)
+   * [Collectd exporter](https://github.com/prometheus/collectd_exporter) (*official*)
+   * [Graphite exporter](https://github.com/prometheus/graphite_exporter) (*official*)
    * [Heka dashboard exporter](https://github.com/docker-infra/heka_exporter)
    * [Heka exporter](https://github.com/imgix/heka_exporter)
    * [Munin exporter](https://github.com/pvdh/munin_exporter)
    * [New Relic exporter](https://github.com/jfindley/newrelic_exporter)
+   * [scollector exporter](https://github.com/tgulacsi/prometheus_scollector)
+   * [StatsD exporter](https://github.com/prometheus/statsd_exporter) (*official*)
+
 
 ### Miscellaneous
    * [BIG-IP exporter](https://github.com/ExpressenAB/bigip_exporter)
    * [BIND exporter](https://github.com/digitalocean/bind_exporter)
+   * [Blackbox exporter](https://github.com/prometheus/blackbox_exporter) (*official*)
    * [Jenkins exporter](https://github.com/lovoo/jenkins_exporter)
+   * [JMX exporter](https://github.com/prometheus/jmx_exporter) (*official*)
```

Comment:
I don't get the context, could you elaborate to which of your comments the "Dito" is referring?